MIGRATING.md

```diff
@@ -1,5 +1,14 @@
 # Migration Guides
 
+## v0.14
+
+- `@theme-ui/components` and other Theme UI packagss no longe depends on
```

Comment:
```suggestion
- `@theme-ui/components` and other Theme UI packages no longer depend on
```

Reply:
Oh god, I made so many mistakes in that sentence xD